lib/menu.rb

```diff
@@ -0,0 +1,15 @@
+class Menu
+
+  attr_accessor :dishes
+
+  def initialize
+    @dishes = {}
+  end  
+
+  def load
```

Comment:
Good use of a has here to store your menu! I think you could actually have popped the hash itself directly under the 'initialize' method then in your 'load' method, you could have written a simpler line which iterates over each 'pair' in the hash and 'puts' them :)